### PR TITLE
Import protoc_plugin history

### PR DESCRIPTION
**Pull request by sigurdm**
And set it up as a mono_pkg.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Protobuf support for dart
 
+[![Build Status](https://travis-ci.org/dart-lang/protobuf.svg?branch=master)](https://travis-ci.org/dart-lang/protobuf)
+
 This repository is home to packages related to support for the protobuf format for dart.
 It contains:
 
-  [protobuf](protobuf/) the support library for generated code.
+  [protoc_plugin](protoc_plugin/) A dart back-end for the protoc compiler.
```
Comment (szakarias) [inline README.md]: Make as list

File: protoc_plugin/build.yaml [hunk 2]
```diff
@@ -1,20 +0,0 @@
-# Read about `build.yaml` at https://pub.dartlang.org/packages/build_config
```
Comment (szakarias) [inline protoc_plugin/build.yaml]: Are you sure this should be deleted?

File: README.md (hunk 1 above)
Comment (sigurdm) [reply README.md]: Done

**Comment by sigurdm** [comment]
This is import of code from dart-protoc-plugin where all commits have CLAs.
Therefore I override the CLA  bot

**Pull request merged**